mod/virtual/lib/card/virtual.rb

```diff
@@ -19,11 +19,13 @@ def fetch card
 
       def save card
         virt = find_by_card card
-        virt ? virt.update(card.virtual_content) : create(card)
+        virt ? virt.update(card.virtual_content) : (virt = create card)
+        cache.write card.key, virt
```

Comment:
doh. not updating cache

Reply:
this already fixed but differently in #807

Reply:
ah, right. I prefer yours.